### ISSUE TITLE
Output a state for using in other steps when PR was blocked

### ISSUE THREAD

**Issue by LucasMMota** [yykamei/block-merge-based-on-time#1341]
For example, I want to comment in PR when the PR is blocked only.
Is there something implemented already for outputting the `yykamei/block-merge-based-on-time` state? Otherwise it'd be nice to have, like: `steps.block_check.outputs.is_pr_blocked`

```yaml
      - uses: yykamei/block-merge-based-on-time@main
        id: block_check
        if: steps.check_label.outputs.is_full_refresh == 'true'
        with:
          timezone: America/Sao_Paulo
          base-branches: "(default)"
          commit-status-description-while-blocking: "PRs that need Full-Refresh must be merged on Monday or Thursday"
          prohibited-days-dates: "Sunday, Tuesday, Wednesday, Friday, Saturday"

      - name: Comment in PR
        if: steps.block_check.outputs.is_pr_blocked == 'true'
        uses: "actions/github-script@v6"
        with:
          script: |
            const comment = `:warning: **PR is blocked** `;
            github.rest.issues.createComment({
              issue_number: context.issue.number,
              owner: context.repo.owner,
              repo: context.repo.repo,
              body: comment
            })
```

**Comment by yykamei** [comment]
Thank you for suggesting such a great idea! That's very interesting.

This action works in two modes: [handlePull](https://github.com/yykamei/block-merge-based-on-time/blob/429fe8e4ad59176c29b0b00409d42ea7bea80ed9/src/run.ts#L62C16-L62C26) and [handleAllPulls](https://github.com/yykamei/block-merge-based-on-time/blob/429fe8e4ad59176c29b0b00409d42ea7bea80ed9/src/run.ts#L23C16-L23C30).

It could be simple in `handlePull`. If a pull request is blocked as it's opened during a blocking time, this action should simply output `is_pr_blocked=true`.

On the other hand, `handleAllPulls`, which runs on "schedule" or "workflow_dispatch" events, determines whether **each** pull request should be blocked or not. Even during a blocking time, some pull requests can be exempted from being blocked with the _no-block-label_ option, which is "no-block" by default. In this case, `is_pr_blocked` is not precise because:

* all pull requests are not always blocked
* multiple pull requests can be blocked; "is" is not used for plurals

However, I like this idea, so let's refine it more 💪

**Comment by yykamei** [comment]
Hi @LucasMMota,

Thank you for reporting this issue. It might be too late, but I finally shipped this feature with #1565.

I changed a bit from your suggestion:

* define the output parameter as `pr-blocked` instead of `is_pr_blocked`
    - As far as I see, GitHub's official actions use kebab-case, and I thought `is_` can be omitted.

I hope you enjoy this Action!
